lib/data/multipleBackendGateway.js

```diff
@@ -62,7 +63,28 @@ const multipleBackendGateway = {
         return client.delete(objectGetInfo, reqUids, callback);
     },
 
-    // checkHealth:
+    healthcheck: (log, callback) => {
+        const multBackendResp = {};
+        async.each(Object.keys(clients), (location, cb) => {
+            const client = clients[location];
+            if (client.clientType && client.clientType === 'scality') {
```

Comment:
Is the check for `client.clientType` necessary? Would it be possible for `client` to be `undefined`?